tests/proc/tproc_untyped_arg.nim

```diff
@@ -6,3 +6,4 @@ discard """
 
 template something(op: proc (v: untyped): void): void =
```

Comment:
by removing the knownIssues:, i get
```
Expected:
'untyped' is only allowed in templates and macros or magic procs

Gotten:
return type 'untyped' is only valid for macros and templates
```

Is this the message that is supposed to be raised? If so , can we enable again?

Reply:
No, I don't think that makes sense. Can you check the original bug report?

The expected says the proc param shouldn't be untyped, while the actual is taking about return types.